media-libs/libgsf/libgsf-1.14.47.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY=" The G Structured File Library"
+DESCRIPTION=" The project aims to provide an efficient extensible i/o abstraction for \
+dealing with different structured file formats."
+HOMEPAGE="https://github.com/GNOME/libgsf"
+COPYRIGHT="2002-2020 Jody Goldberg"
+LICENSE="GNU LGPL v2.1"
```

Comment:
Regular COPYING file mentions GNU GPL v2, maybe add it here also?

Reply:
added